### PR TITLE
Add syslog-facility option

### PR DESCRIPTION
**Pull request by stroop23**
This PR adds an option to set the syslog facility.

### CODE REVIEW

**Comment by phemmer** [comment]
I would recommend more user friendly values, such as `daemon` and `local0` (instead of `LOG_DAEMON` and `LOG_LOCAL0`).

**Comment by d23** [comment]
I chose not to do that, because now it's not needed to replicate the list in the documentation. It does make sense to have nicer names though. Should I then add a list of supported facilities in the documenation, or refer to, say the syslog rfc?

**Comment by d23** [comment]
Updated for nicer names and also integer facilities.

**Comment by runcom** [comment]
Sounds good to me, I think I can move this to code review ping @LK4D4

**Comment by LK4D4** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

**Comment by runcom** [comment]
pign @thaJeztah @moxiegirl

**Comment by LK4D4** [comment]
ping @thaJeztah @moxiegirl :)

**Comment by thaJeztah** [comment]
thanks @LK4D4, this one slipped my mind. I'll look at this later today

File: docs/sources/reference/run.md [hunk 1]
```diff
@@ -894,7 +895,12 @@ remote server at `192.168.0.42` on port `123`
 
     $ docker run --log-driver=syslog --log-opt syslog-address=tcp://192.168.0.42:123
 
-`syslog-tag` specifies tag for syslog messages from container.
+`syslog-facility` specifies the syslog facility. This can be an integer of 0 to 23 or any of the
+following named facilities: kern, user, mail, daemon, auth, syslog, lpr, news, uucp, cron,
+authpriv, ftp, local0, local1, local2, local3, local4, local5, local6, local7
```
Comment (thaJeztah) [inline docs/sources/reference/run.md]: I doubted if the options should be in back-ticks, but that may become very "busy" (layout-wise), so I suggest leaving it as-is (we can always revisit later)

@moxiegirl wdyt?

Comment (thaJeztah) [inline docs/sources/reference/run.md]: Please add:

```
If no `--syslog-facility` option is provided, the 'daemon' facility is used as default.
```

Comment (thaJeztah) [inline docs/sources/reference/run.md]: (sorry for the noise; edited twice, because I wasn't sure if line 887 was an *example* or the default value)

**Comment by thaJeztah** [comment]
Thanks! I added a suggestion to add to the docs.

File: docs/sources/reference/run.md (hunk 1 above)
Comment (spil-dennis) [inline docs/sources/reference/run.md]: Will fix, i actually had it in there earlier.

**Comment by thaJeztah** [comment]
Thanks for addressing; LGTM

ping @fredlf @jamtur01 @moxiegirl @SvenDowideit

**Comment by runcom** [comment]
@spil-dennis sorry this needs a rebase :/

**Comment by d23** [comment]
Hehe sure np

**Comment by runcom** [comment]
ping @moxiegirl

File: docs/reference/run.md [hunk 2]
```diff
@@ -901,7 +902,13 @@ remote server at `192.168.0.42` on port `123`
 
     $ docker run --log-driver=syslog --log-opt syslog-address=tcp://192.168.0.42:123
 
-`syslog-tag` specifies tag for syslog messages from container.
+`syslog-facility` specifies the syslog facility. This can be an integer of 0 to 23 or any of the
```
Comment (moxiegirl) [inline docs/reference/run.md]: This is really hard to read without the list.  Also, even though we provide bad examples earlier, should't be starting a sentence with the lowercase command.  

--> https://gist.github.com/moxiegirl/abe4816bb529908fae92

The `syslog-facility` option configures the syslog facility. By default, the system uses the `daemon` value. To override this behavior, you can provide an integer of 0 to 23 or any of the following named facilities: 

* `kern`
* `user`
* `mail`
* `daemon`
* `auth`
* `syslog`
* `lpr`
* `news`
* `uucp`
* `cron`
* `authpriv`
* `ftp`
* `local0`
* `local1`
* `local2`
* `local3`
* `local4`
* `local5`
* `local6`
* `local7`

The `syslog-tag` specifies tag for syslog messages from container. By default, the system uses the first 12 characters of the container id. To override this behavior, specify a `syslog-tag` option

**Comment by spil-dennis** [comment]
@moxiegirl Thanks for the review, updated as per your suggestions.

**Comment by d23** [comment]
@moxiegirl Thanks for the review, updated as per your suggestions.

**Comment by moxiegirl** [comment]
@spil-dennis Thanks for the contribution and the quick response.  Very sorry for the delay in processing this.  ping @LK4D4 moving to merge status.

**Pull request merged**